tests/run-make/rust-lld-custom-target/rmake.rs

```diff
@@ -15,7 +15,7 @@ fn main() {
     // Compile to a custom target spec with rust-lld enabled by default. We'll check that by asking
     // the linker to display its version number with a link-arg.
     let output = rustc()
-        .env("RUSTC_LOG", "rustc_codegen_ssa::back::link=info")
+        .verbose()
```

Comment:
You are using `.verbose()` here, but `.arg("--verbose")` elsewhere. Can those `.arg("--verbose")` be replaced with `.verbose()`?

Reply:
good idea, done